src/main.py

```diff
@@ -64,17 +70,28 @@ def task1(conn):
 
       # let's be kind to the user and not make it case sensitive
 
-      if option.lower() == "n" and showing < batches:
-        # next page
-        showing += 1
-        clear()
-        continue
+      if option.lower() == "n":
+        if showing < batches:
+            # next page
+            showing += 1
+            clear()
+            continue
+        else:
+          # no next page
+          print("\nThere are no more pages. Enter P to go back a page, or choose a paper number.")
+          continue
     
-      if option.lower() == "p" and showing >1:
-        # prev page
-        showing -= 1
-        clear()
-        continue
+      if option.lower() == "p":
+        if showing >1:
+          # prev page
+          showing -= 1
+          clear()
+          continue
+        else:
+          # no prev page
+          print("\nThere are no previous pages. Enter N to go to the next page, or choose a paper number.")
```

Comment:
Why do you not clear in the event that the page goes out of bounds? Visually, not doing so can defeat the purpose of having the clear at all.

Reply:
Ues, it's intentional. Clearing the screen would make the error message disappear too fast to see. How about I add a time delay using time.sleep, and then clear the screen?

Reply:
Actually, I think it's better to change it to the delay; it looks a lot nicer.